packages/cli/test/unit/utils.unit.js

```diff
@@ -105,22 +119,95 @@ describe('Utils', () => {
         'ÓooBar',
       );
     });
+
     function testExpectError(testName, input, expected) {
       it(testName, () => {
         expect(utils.toClassName(input)).to.match(expected);
       });
     }
+
     function testExpectNoChange(testName, input) {
       it(testName, () => {
         expect(utils.toClassName(input)).to.equal(input);
       });
     }
+
     function testExpectUpperCase(testName, input, expected) {
       it(testName, () => {
         expect(utils.toClassName(input)).to.equal(expected);
       });
     }
   });
+
+  describe('validateUrlSlug', () => {
```

Comment:
With the way `validateUrlSlug` is currently coded, the function accepts camel casings along with other words separated by url-safe separators (such as _). Should we potentially look to enforce kebab-casing?